clang/unittests/Interpreter/InterpreterTest.cpp

```diff
@@ -91,6 +93,53 @@ TEST_F(InterpreterTest, IncrementalInputTopLevelDecls) {
   EXPECT_EQ("var2", DeclToString(*R2DeclRange.begin()));
 }
 
+TEST_F(InterpreterTest, BadIncludeDoesNotCorruptTU) {
```

Comment:
Can we extend either `clang/test/Interpreter/execute.c` or `clang/test/Interpreter/fail.cpp` tests instead? I think it would be simpler to write and maintain.